app/store/message.go

```diff
@@ -118,11 +118,16 @@ func LoadMessagesFromDB() error {
 	}
 	return nil
 }
+
 func DeleteMessage(id int64) error {
-	// TODO: delete Attachments
-	_, err := DS.Dbx.Exec("DELETE FROM messages WHERE id = ?", id)
+	err := deleteAttachmentForMessage(id)
+	if err!=nil{
+		log.Errorln("[axolotl] could not delete attachment", err)
+	}
+	_, err = DS.Dbx.Exec("DELETE FROM messages WHERE id = ?", id)
```

Comment:
Do we want to continue to delete the message if the deletion of the attachment failed?

Reply:
When we delete the message, we loose all reference to the attachment, so maybe not

Reply:
Yeah so let's completely fail here please.